templates/templates/navigation-enterprise-h.html

```diff
@@ -198,6 +194,7 @@ <h4 class="p-muted-heading--small">Fully managed</h4>
           <li class="p-inline-list__item"><a href="/kubernetes/managed">Kubernetes</a></li>
           <li class="p-inline-list__item"><a href="/managed-apps">Managed apps</a></li>
           <li class="p-inline-list__item"><a href="https://anbox-cloud.io/" title="Visit Anbox Cloud">Anbox Cloud</a></li>
+          <li class="p-inline-list__item"><a href="/smartstart">Smart Start for IoT</a></li>
```

Comment:
```suggestion
          <li class="p-inline-list__item"><a href="/core/smartstart">Smart Start for IoT</a></li>
```

this moved with UC20